jobs/routing-api/spec

```diff
@@ -108,3 +108,7 @@ properties:
   routing_api.lock_retry_interval:
     description: "interval to wait before retrying a failed lock acquisition"
     default: "5s"
+
+  routing_api.listen_ip:
+    description: "IP for routing api to listen on"
+    default: "0.0.0.0"
```

Comment:
Why is the default for this property different from the default for `haproxy.health_check_ip`?